### DISCUSSION TITLE
Navigate to recent files

### DISCUSSION THREAD

**Discussion by QazPolma** [godotengine/godot-proposals#5802] [Editor]
An option to be able to "ctrl + tab" to previous files, like in IntelliJ, would be nice.

While going to method definition is done with "ctrl + mouse click" or global search for method name, going backwards relies on "alt + left arrow" or having to recall which file I started from. And it feels distracting when I have to click files with mouse when I constantly jump between few recent files while analyzing method usage and implementation of those methods.

**Comment by oeleo1** [comment]
Come on :-) If you are an IntelliJ user you don’t have a problem with the Godot editor goodies at all. You have a back arrow going through the history of your CTRL-clicks.

**Comment by QazPolma** [comment]
When using recent files function I can keep a "bookmark" in all open files. With the back function I'd lose the "bookmark" as soon as I go back and take a different forward branch.
